src/Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -1454,12 +1473,221 @@ public override void DeleteNextStateRootHash(BlockHash blockHash)
             }
         }
 
+        /// <inheritdoc/>
+        public override IEnumerable<EvidenceId> IteratePendingEvidenceIds()
+        {
+            foreach (Iterator it in IterateDb(_pendingEvidenceDb, PendingEvidenceKeyPrefix))
+            {
+                byte[] key = it.Key();
+                byte[] idBytes = key.Skip(PendingEvidenceKeyPrefix.Length).ToArray();
+                yield return new EvidenceId(idBytes);
+            }
```

Comment:
Can we use [prefix seek](https://github.com/facebook/rocksdb/wiki/Prefix-Seek) for performance?

Reply:
The store-related code was implemented by @OnedgeLee, so I have no understanding of this part. It will take some time to improve it.

Reply:
This issue has been postponed.